servers/physics_2d/godot_body_pair_2d.h

```diff
@@ -62,13 +62,14 @@ class GodotBodyPair2D : public GodotConstraint2D {
 		real_t acc_normal_impulse = 0.0; // accumulated normal impulse (Pn)
 		real_t acc_tangent_impulse = 0.0; // accumulated tangent impulse (Pt)
 		real_t acc_bias_impulse = 0.0; // accumulated normal impulse for position bias (Pnb)
+		real_t acc_bias_impulse_center_of_mass = 0.0; // accumulated normal impulse for position bias applied to com
```

Comment:
Shouldn't inline comments also start with a capital letter and end with a point?

Reply:
Yeah these comments should follow the same rules, but for now I've just copied it the same way. Physics servers have more styling issues like that and will require a pass to fix that at some point :)